newIDE/app/src/ProjectsStorage/LocalFileStorageProvider/LocalProjectWriter.js

```diff
@@ -24,6 +24,32 @@ export const splittedProjectFolderNames = [
   'eventsFunctionsExtensions',
 ];
 
+const deleteExistingFilesFromDirs = (projectPath: string) => {
+  const entries = fs.readdirSync(projectPath);
+  return new Promise((resolve, reject) => {
```

Comment:
You don't need to use promises here, all the filesystem methods you use are synchronous.

Reply:
(I think `fs.unlinkSync` also exists)